app/models/resource.rb

```diff
@@ -105,6 +105,9 @@ def search account: nil, kind: nil, owner: nil, offset: nil, limit: nil, search:
       scope = scope.textsearch(search) if search
 
       if offset || limit
+        if (offset && !numeric?(offset)) || (limit && !numeric?(limit))
+          raise ApplicationController::UnprocessableEntity, "Delimiter must be an integer if given"
```

Comment:
@shulifink can you please review this message?

Reply:
I don't understand. What do you mean "if given"?

Reply:
@liavyona Isn't it enough to just say "If you provide a value for Delimiter, it must be an integer greater than or equal to 0".

or simply: "Delimiter must be an integer greater than or equal to 0".

or  "... must be an integer", or "must be a positive integer " (depending on what the integer type)

Reply:
I would go with "Delimiter offset or limit must be an integer greater than or equal to 0"